### PR TITLE
update MAP and scdbg

### PR DESCRIPTION
**Pull request by mr-tz**
supersedes #601

### CODE REVIEW

File: packages/map.vm/map.vm.nuspec
Comment (Ana06) [inline packages/map.vm/map.vm.nuspec:5]: We should always increase the version, otherwise the old version will be installed (and tested). We need to add this to the linter. We should use the tool version here (and not the date). If there is no tool version, the version should be 0.0.0.DATE, using the 4th segment for the date. This way we ensure we always increase the version.

As the new version will be probably lower after using the correct one, before we can merge this PR, we need to change the version, delete the package in myget and run the CI to ensure this new version is working. We should try to merge this PR fast after deleting the package in myget as it will fail to install during that time.

**Review by Ana06 — CHANGES_REQUESTED** [review]

**Comment by mr-tz** [comment]
If you want, please go ahead and take this.

**Review by Ana06 — APPROVED** [review]
I have deleted the packages in myget and updated the version. The test suite is now testing the correct version. I am going to merge this quick, as the packages can't be installed at the moment.

**Pull request merged**
